x-pack/plugin/sql/jdbc/qa/build.gradle

```diff
@@ -0,0 +1,66 @@
+description = 'Integration tests for SQL JDBC'
+apply plugin: 'elasticsearch.build'
+archivesBaseName = 'qa-sql-jdbc'
+group = "org.elasticsearch.x-pack.qa.sql.jdbc"
+
+dependencies {
+    compile project(":test:framework")
+
+    // JDBC testing dependencies
+    compile project(path: xpackModule('sql:jdbc'))
+}
+
+/* disable unit tests because these are all integration tests used
+ * other qa projects. */
+test.enabled = false
+
+dependencyLicenses.enabled = false
+dependenciesInfo.enabled = false
+
+// the main files are actually test files, so use the appropriate forbidden api sigs
+tasks.named('forbiddenApisMain').configure {
+    replaceSignatureFiles 'es-all-signatures', 'es-test-signatures'
+}
+
+// just a test fixture: we aren't using this jars in releases and H2GIS requires disabling a lot of checks
+thirdPartyAudit.enabled = false
+
+subprojects {
+    if (subprojects.isEmpty()) {
+        // leaf project
+        apply plugin: 'elasticsearch.standalone-rest-test'
+    } else {
+        apply plugin: 'elasticsearch.build'
```

Comment:
We should use `elasticsearch.java` plugin again here. The nested qa projects don't need all that additional precommit stuff.